hs_core/templates/pages/genericresource.html

```diff
@@ -850,6 +853,28 @@ <h4 class="modal-title" id="myModalLabel">Add file to resource</h4>
                                 $res_title.text(updated_title);
                             }
                         }
+
+                        if (json_response.hasOwnProperty('metadata_status')){
+                            if(json_response.metadata_status !== $('#metadata-status').text()){
+                                $('#metadata-status').text(json_response.metadata_status);
+                                if(json_response.metadata_status == "Sufficient to make public")
+                                {
+                                    alert("Metadata Status:Sufficient to make public");
```

Comment:
I didn't mean it to be debug code. Is there a better way to notify the user? I am not a JS guru.